.github/workflows/publish-docker-containers.yml

```diff
@@ -61,7 +62,7 @@ jobs:
         uses: docker/setup-buildx-action@v2
 
       - name: Login to DockerHub
-        if: ${{ inputs.publish || startsWith(github.ref, 'refs/tags/') }}
+        # if: ${{ inputs.publish || startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Why we need login for dockerhub when we are publishing to github repo?

Reply:
I changed to login to ghcr.io instead